adapters/repos/db/helpers/tokenizer.go

```diff
@@ -91,6 +155,17 @@ func tokenizeWordWithWildcards(in string) []string {
 	return lowercase(terms)
 }
 
+// tokenizetrigram splits on any non-alphanumerical and lowercases the words, joins them together, then groups them into trigrams
+func tokenizetrigramWithWildcards(in string) []string {
```

Comment:
Was this comment accidentally lifted from the `tokenizetrigram` function header? Looks the same, and this comment doesn't explain the wildcard part

Reply:
yes, thanks for spotting that